services/app/src/static/style/header.css

```diff
@@ -0,0 +1,54 @@
+#header {
+  width: 100%;
+  height: 80px;
+  top: 0%;
+  left: 0%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background: #f3f6fa;
+  position: absolute;
+  flex-grow: 1;
+  min-width: 800px;
+}
+#logo {
+  margin-left: 64px;
+}
+#seperator {
+  margin-left: 16.5px;
+  margin-right: 16.5px;
+}
+#roomName {
+  margin-right: 18px;
+}
+#copyLink {
+  margin-right: 20.28px;
+}
+#settings {
```

Comment:
`settings` has the same stying as `copyLink`. If I'm not mistaken, you are describing top menu navigation items here. If this is the case, we should come up with some common class name and use it for all the navigation items.

By the way, I've noticed you are using css `id`s extensively. I think we should follow common approach and use `id` when we are describing item that needs to be uniquely identified and fallback to using css `class` in all the other cases.

Reply:
For some reason, gap does not work in this case, so I had to use margin-right instead

Reply:
Might be something related with `display` property. Nevertheless, I checked out the branch and did some css ground work. You can check it out on `feature/header-suggestions` branch.

One thing I did not address on that branch is css file locations. If we have a component (e.g. Header) it is really handy to have `.css` files that contain classes for this component be right next to it in the file tree. At the moment all the css is located in `static/style`, but I'd like to have styles for header inside `components/header/styles/header.css`.

I'm not sure how webpack config will like this setup, but please try if it works and let me know.